changelog/1108.bugfix.rst

```diff
@@ -0,0 +1 @@
+Make message_command check both local and synchronized guild_ids
```

Comment:
This PR fixes multiple things, not only the `message_command` decorator. I think it'd be better to say which bug has been fixed.